src/InvvardDev.EZLayoutDisplay.Tests/Service/LayoutServiceTest.cs

```diff
@@ -14,27 +14,29 @@ public class LayoutServiceTest
     {
```

Comment:
VS decided to get a little aggressive with the auto-formatting here, I'll revert it.